### PR TITLE
Navi 40/create-albums-page

### PR DESCRIPTION
**Pull request by Nikushha34**

### CODE REVIEW

File: app/album/page.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+'use client'
+
+import styles from "./page.module.scss";
+import { useEffect } from "react";
+import Albumsssv2 from "../components/Albumsssv2/page";
```
Comment (lursmana1) [inline app/album/page.tsx]: change name of this

File: app/components/Albums/AlbumCard/AlbumCard.module.scss [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    }
+
+    .spans {
+        display: flex;
+        justify-content: space-between;
+        max-width: 512px;
+    }
+}
+
+@media screen and (min-width: 1920px) {
```
Comment (lursmana1) [inline app/components/Albums/AlbumCard/AlbumCard.module.scss]: u have same media above

File: app/components/Albums/Albums.module.scss [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+}
+
+@media screen and (min-width: 1024px) {
+    .Search {
+        width: 607px;
+    }
+
+}
+
+@media screen and (min-width: 1920px) {
```
Comment (lursmana1) [inline app/components/Albums/Albums.module.scss]: same

File: app/components/Albums/Albums.tsx [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+import Link from "next/link";
+import AlbumCard from "./AlbumCard/AlbumCard";
+import { Search } from "../Header/Search/Search";
+import { albumsState } from "@/app/states";
+
+
+
+export const Albums = () => {
+
+    const [albums] = useRecoilState(albumsState)
```
Comment (lursmana1) [inline app/components/Albums/Albums.tsx]: u are not using this

File: app/components/Albumsssv2/Albumsartists/page.module.scss [hunk 5]
```diff
@@ -0,0 +1,196 @@
...
+    }
+
+    .wrapper {
+        width: 320px;
+        margin-left: 10px;
+    }
+}
+
+
+@media (width: 540px) {
```
Comment (lursmana1) [inline app/components/Albumsssv2/Albumsartists/page.module.scss]: this whole css needs refactor why 5 break poinsts ?

File: app/components/Albumsssv2/Songs/page.tsx [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+import Albumsartistsv2 from "../Albumsartists/page";
+import { useSearchParams } from "next/navigation";
+
+
+const AlbumSongs = () => {
+    let params = useSearchParams();
+    let albumParam = params.get('album');
+
+    const [albumName, setAlbumName] = useState('')
+    const [myAlbum, setMyAlbum] = useRecoilState(albumsState)
```
Comment (lursmana1) [inline app/components/Albumsssv2/Songs/page.tsx]: is recoil surely needed here , i dont think soo

File: app/components/Albumsssv2/page.module.scss [hunk 7]
```diff
@@ -0,0 +1,238 @@
...
+
+    .SearchBar {
+        display: flex;
+        justify-content: center;
+        width: 650px;
+
+    }
+}
+
+@media screen and (max-width: 1837px) {
```
Comment (lursmana1) [inline app/components/Albumsssv2/page.module.scss]: duplication again

File: app/components/Albumsssv2/Songs/page.tsx (hunk 6 above)
Comment (Nikushha34) [reply app/components/Albumsssv2/Songs/page.tsx]: yes, we need this

**Pull request merged**